Formula/diesel.rb

```diff
@@ -0,0 +1,32 @@
+class Diesel < Formula
+  desc "Command-line tool for Rust ORM Diesel"
+  homepage "https://diesel.rs"
+  url "https://github.com/diesel-rs/diesel/archive/v1.4.3.tar.gz"
+  sha256 "79769964028e5cc85219d100d4945d69a5d2ec5a4125c36fd50b9ef1dc326d5a"
+
+  head "https://github.com/diesel-rs/diesel.git"
+
+  depends_on "rust" => [:build, :test]
+  depends_on "libpq"
+  depends_on "mysql-client"
+  uses_from_macos "sqlite"
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "diesel_cli"
```

Comment:
can you add `--locked` flag? relates to https://github.com/Homebrew/homebrew-core/issues/46025

Reply:
There seems to be no `Cargo.lock` file in Diesel's repository.

Reply:
Looks like it relates to this issue, https://github.com/diesel-rs/diesel/issues/2209

Historically, it does have `cargo.lock` in the repo though, not sure when and why it got removed.
https://github.com/diesel-rs/diesel/issues/239